contracts/margin/external/ERC20/ERC20CappedLong.sol

```diff
@@ -72,4 +98,15 @@ contract ERC20CappedLong is ERC20Long {
 
         return tokenAmount;
     }
+
+    function setTokenCapInternal(
```

Comment:
Is there a reason you have this as a private function and don't just put those two lines in `setTokenCap`?

Reply:
It's in the constructor too